Jenkinsfile

```diff
@@ -31,5 +31,12 @@ ansiColor('xterm') {
         archive includes: "*log.tar.gz"
       }
     }
+
+    stage('Release unstable MoM EE Docker Image') {
+      when { branch: 'master' }
+      steps {
+        build job: '/Marathon/job/marathon-dcos-plugins/job/release-mom-ee-docker-image/job/master/', parameters: [string(name: 'from_image_tag', value: 'unstable')], propagate: true
```

Comment:
I still need to modify the  `marathon-dcos-plugins` Docker job to release the master as `unstable`.

Reply:
See https://github.com/mesosphere/marathon-dcos-plugins/pull/203.